space-template.yaml

```diff
@@ -0,0 +1,172 @@
+---
+# These define the potential field types that ca be used in the fields of work
+# item types.
+field_types:
+  - &string string
+  - &translatable_string translatable_string
+  - &list list
+  - &timestamp timestamp
+  - &user user
+
```

Comment:
These are the simple/basic field types. All known by the system, user can not add their own.